### PR TITLE
Update/data add tissues

### PR DESCRIPTION
**Pull request by biodavidjm**
Added two new rat tissues, and updated the related data objects

### CODE REVIEW

File: inst/extdata/bic_animal_tissue_code.tsv [hunk 1]
```diff
@@ -21,7 +21,7 @@ T19	Human Mycoplasma	T19 Human Mycoplasma	t19-mycoplasma
 T20	Human Muscle Histology	T20 Human Muscle Histology	t20-muscle-histology
 T21	Human Adipose Histology	T21 Human Adipose Histology	t21-adipose-histology
 T30	Blood RNA	T30 Rat PaxGene RNA	t30-blood-rna
-T31	EDTA Plasma	T31 Rat Plasma	t31-plasma
```
Comment (nicolerg) [inline inst/extdata/bic_animal_tissue_code.tsv:24]: why remove this tissue code? isn't it still used in the PASS data releases?

File: inst/extdata/bic_animal_tissue_code.tsv [hunk 2]
```diff
@@ -60,4 +60,6 @@ T66	Lung Powder	T66 Rat Lung Powder	t66-lung
 T67	Small Intestine Powder	T67 Rat Small Intestine Powder	t67-small-intestine
 T68	Liver Powder	T68 Rat Liver Powder	t68-liver
 T69	Brown Adipose Powder	T69 Rat Brown Adipose Powder	t69-brown-adipose
-T70	White Adipose Powder	T70 Rat White Adipose Powder	t70-white-adipose
\ No newline at end of file
+T70	White Adipose Powder	T70 Rat White Adipose Powder	t70-white-adipose
+T77	Lateral Gastrocnemius	T77 Rat Lateral Gastrocnemius	t77-lateral-gastroc
```
Comment (nicolerg) [inline inst/extdata/bic_animal_tissue_code.tsv:64]: Should it be "Lateral Gastrocnemius Powder" and "Vena Cava Powder" for consistency with all the other rat tissues,?

File: inst/extdata/motrpac_colors_abbr.R [hunk 3]
```diff
@@ -150,81 +156,86 @@ tissue_cols = c('paxgene rna'='#d92c04', # blood and paxgene rna are the same (R
...
                 gastrocnemius='#088c03',
                 'SKM-GN'='#088c03',
-                "t55-gastrocnemius"='#088c03',
+                't55-gastrocnemius'='#088c03',
                 'vastus lateralis'='#025939',
                 'vastus-lateralis'='#025939',
                 'vastus_lateralis'='#025939',
                 'SKM-VL'='#025939',
-                "t56-vastus-lateralis"='#025939',
+                lateralgastroc = '#025939',
```
Comment (nicolerg) [inline inst/extdata/motrpac_colors_abbr.R:186]: this should be '#025950', not '#025939'. would suggest moving it to the line below so lines are grouped by tissue

File: inst/extdata/tissue_codes.R [hunk 4]
```diff
@@ -2,6 +2,7 @@
 # R script to generate the tissue codes and colors
 library(data.table)
 library(devtools)
+library(MotrpacBicQC)
```
Comment (nicolerg) [inline inst/extdata/tissue_codes.R:5]: usually better to use `devtools::load_all()` than loading the library if you are changing data in the library

**Review by nicolerg — CHANGES_REQUESTED** [review]

File: inst/extdata/bic_animal_tissue_code.tsv (hunk 1 above)
Comment (biodavidjm) [reply inst/extdata/bic_animal_tissue_code.tsv:24]: oh, nice catch. I downloaded a copy from where we keep the source of truth, but for whatever reason that was missed there. I put it back. Thanks!

File: inst/extdata/bic_animal_tissue_code.tsv (hunk 2 above)
Comment (biodavidjm) [reply inst/extdata/bic_animal_tissue_code.tsv:64]: nice catch again! Thanks!

File: inst/extdata/motrpac_colors_abbr.R (hunk 3 above)
Comment (biodavidjm) [reply inst/extdata/motrpac_colors_abbr.R:186]: Great! I will do all that. Thanks a lot!

File: inst/extdata/tissue_codes.R (hunk 4 above)
Comment (biodavidjm) [reply inst/extdata/tissue_codes.R:5]: You are right. And that's how I've always done it but I decided this time to load the library instead in that way. I will go back. Thanks for pointing this out

**Pull request merged**